pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -71,15 +80,29 @@ def convert_to_nwb(info: dict) -> bool:
     Function used to convert the source data to NWB format using the specified metadata.
     """
 
+    nwbfile_path = info["nwbfile_path"]
+
+    runStubTest = ("stub_test" in info) and info["stub_test"]
```

Comment:
```suggestion
    runStubTest =  info.get("stub_test")
```
Dictionaries in Python have a 'safe' retrieval functionality through `.get(key, default)`

`default` is `None` by, well, 'default'; but `if` logic treats `None` the same as `False` so works as is with the logic below